run_server.sh

```diff
@@ -17,11 +17,23 @@
 TEAM_ID="$1"
 TEAM_SECRET="$2"
 PORT="$3"
+RELAY_ADDRESS="$4"
 
 if [[ "TEAM_ID" == "" || "$TEAM_SECRET" == "" || "$PORT" == "" ]] ; then
-  echo 'usage: <TEAM ID> <TEAM SECRET> <PORT>'
+  echo 'usage: <TEAM ID> <TEAM SECRET> <PORT> [RELAY ADDRESS]'
   exit 1
 fi
 
 cd './bin'
-java codeu.chat.ServerMain "$TEAM_ID" "$TEAM_SECRET" "$PORT"
+if [ "$RELAY_ADDRESS" == "" ] ; then
```

Comment:
Yes. If I don't do this, an empty argument will get passed to the java run time and then it will try to parse that as an address and crash.